README.md

```diff
@@ -1,10 +1,11 @@
-- [Linux_for_Tegra](#Linux_for_Tegra)
-    - [Supported hardware](#supported-hardware)
-    - [Getting Started](#getting-started)
-    - [Introduction to the software](#introduction-to-the-software)
-        - [Directory structure](directory-structure)
-        - [CI/CD](#cicd)
-    - [Summary](#summary)
+- [Linux\_for\_Tegra](#linux_for_tegra)
+  - [Supported hardware](#supported-hardware)
```

Comment:
Why you change 4 spaces to 2 spaces?

Reply:
I use the plugin https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one probably formatted the document for me this way, this applies to both indentation and "\" characters. The decision is yours, if you want I can make another PR and apply the previous formatting style there

Reply:
There is no need to make this change.So please remove it.